source/databases-collections.txt

```diff
@@ -34,38 +34,35 @@ use the left navigation to:
 Databases and Collections
 -------------------------
 
-When you expand an active connection, |vsce| shows the databases in that
-deployment. Click a database to view the collections it contains.
+When you expand an active connection, |vsce| shows the databases in
+that deployment. Click a database to view the collections it contains.
 
 Click a collection to view its documents, schema, and indexes.
 
+.. include:: /includes/admonitions/close-all-docs-on-exit.rst
+
 View Collection Documents and Schema
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
[Suggestion]
I would move the sentence below in this comment and place it right under the "View Collection Documents and Schema" heading. I would edit the sentence a bit too:

When you expand a collection, |vsce| displays the number of documents it found next to the :guilabel:`Documents` label in the navigation panel.